release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2557,3 +2557,19 @@ link:https://access.redhat.com/solutions/5866491[{product-title} 4.5.34 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.5 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[id="ocp-4-5-35"]
+=== RHBA-2021:0785 - {product-title} 4.5.35 bug fix update
```

Comment:
Since this includes RHSAs, we should probably list that in the title for clarity (along with the corrected type):
> === RHSA-2021:0785 - {product-title} 4.5.35 bug fix and security update

Reply:
Fixed.